Configuration/TCA/Overrides/sys_file_metadata.php

```diff
@@ -4,12 +4,12 @@
 $tca = array(
 	'types' => array(
 		TYPO3\CMS\Core\Resource\File::FILETYPE_IMAGE => array('showitem' => '
-			fileinfo, title, description, alternative, keywords, caption, download_name,
-
-			--div--;LLL:EXT:cms/locallang_ttc.xml:tabs.access,
-				--palette--;LLL:EXT:filemetadata/Resources/Private/Language/locallang_tca.xlf:palette.visibility;10;;,
+			fileinfo, title, description, alternative, keywords, caption, download_name,' .
+			TRUE === version_compare(TYPO3_version, '7.1', '<') ?
```

Comment:
This code was not working for me. The first tab was completely skipped. Version of PHP 5.6? I went for a more verbose but supported solution even if there is some duplicate code involved. This will be removed when dropping 6.2 compatibility.